deploy-manage/deploy/elastic-cloud/change-hardware.md

```diff
@@ -8,9 +8,11 @@ products:
   - id: cloud-hosted
 ---
 
-# Change hardware [ec-change-hardware-for-a-specific-resource]
+# Customize instance configuration [ec-change-instance-configuration]
```

Comment:
Please add an introduction to the document explaining its scope / purpose, and also explaining when the document can be useful (what use case it addresses).

I would start with something like:

> This document explains how to modify the instance configurations used by specific components of your deployment without changing the overall hardware profile assigned to the deployment. This advanced configuration scenario is useful when ...... (to be determined)

We need to explain `when` such a procedure would be needed by our users.